app/championship/knights.py

```diff
@@ -0,0 +1,51 @@
+class Knight:
+
+    knights = dict()
+
+    def __init__(
+            self,
+            name: str,
+            power: int,
+            hp: int,
+            protection: int
+    ) -> None:
+        self.name = name
+        self.power = power
+        self.hp = hp
+        self.protection = protection
+
+    @classmethod
+    def add_knights(cls, knight_stat: dict) -> None:
+        # apply base stat
+        name = knight_stat["name"]
+        hp = knight_stat["hp"]
+        protection = 0
+        power = knight_stat["power"]
+
+        # apply armour
+        if knight_stat["armour"]:
+            protection = sum(
+                [part["protection"] for part in knight_stat["armour"]]
```

Comment:
You do not need to use list comprehension inside of `sum` here, rewrite code without using it.